mage_ai/frontend/components/FileBrowser/index.tsx

```diff
@@ -474,6 +501,13 @@ function FileBrowser({
             showModalNewFile({ file: selectedFile, moveFile: true });
           },
           uuid: 'move_file',
+        }, {
+          label: () => 'Download file',
+          onClick: () => {
+            const fp = getFullPathWithoutRootFolder(selectedFile);
+            downloadFile(encodeURIComponent(fp));
```

Comment:
Saw a linting error for a missing dependency, so you may need to include `downloadFile` in the useMemo dependency array here: https://github.com/mage-ai/mage-ai/pull/3813/files#diff-5df45198d443d4d572868f9c02c6c2d71a000c912c5a5b254ce42033ae6c930cR568-R581